addons/l10n_bf/data/account_tax_report_data.xml

```diff
@@ -415,7 +415,12 @@
                         <field name="engine">aggregation</field>
                         <field name="formula">BF_DEDUCTIBLE.balance + BF_CREDIT_REPORTED.balance - BF_CREDIT_ASKED.balance + BF_OTHER_DEDUCTION.balance + BF_UNPAID_CREDIT.balance + BF_OTHER_DEDUCTION.balance - BF_GROSS_TOTAL.balance</field>
                         <field name="subformula">if_above(EUR(0))</field>
-                        <field name="carryover_target">BF_REPORTED.balance</field>
+                    </record>
+                    <record id="account_tax_report_line_bf_credit_to_report_carryover" model="account.report.expression">
+                        <field name="label">_carryover_balance</field>
+                        <field name="engine">aggregation</field>
+                        <field name="formula">BF_CREDIT_TO_REPORT.balance</field>
+                        <field name="carryover_target">BF_CREDIT_REPORTED.balance</field>
```

Comment:
Did you test it ? It still seems incomplete to me :thinking:  

The carryover_target field should reference an _applied_carryover_.... expression, just like here https://github.com/odoo/odoo/blob/17.0/addons/l10n_fr/data/tax_report_data.xml#L1333 ; else the amount that got carried over will never be taken into account in the report. 

Of course, to be considered by the report, that expression should also be called by an aggregation, so you'll likely need 3 expressions, like here https://github.com/odoo/odoo/blob/17.0/addons/l10n_fr/data/tax_report_data.xml#L1045


(this comment applies to the other l10n touched by this PR as well)

Reply:
@oco-odoo  Yeah everything worked fine I tested each localization :)